x-pack/legacy/plugins/siem/public/utils/kql/use_update_kql.tsx

```diff
@@ -9,20 +9,16 @@ import { Dispatch } from 'redux';
 import { StaticIndexPattern } from 'ui/index_patterns';
 
 import { KueryFilterQuery } from '../../store';
-import { HostsType } from '../../store/hosts/model';
-import { applyHostsFilterQuery as dispatchApplyHostsFilterQuery } from '../../store/hosts/actions';
-import { applyNetworkFilterQuery as dispatchApplyNetworkFilterQuery } from '../../store/network/actions';
 import { applyKqlFilterQuery as dispatchApplyTimelineFilterQuery } from '../../store/timeline/actions';
 import { convertKueryToElasticSearchQuery } from '../../lib/keury';
 import { RefetchKql } from '../../store/inputs/model';
-import { NetworkType } from '../../store/network/model';
 
 interface UseUpdateKqlProps {
   indexPattern: StaticIndexPattern;
   kueryFilterQuery: KueryFilterQuery | null;
   kueryFilterQueryDraft: KueryFilterQuery | null;
-  storeType: 'networkType' | 'hostsType' | 'timelineType';
-  type: HostsType | NetworkType | null;
+  storeType: 'timelineType';
+  type: null;
```

Comment:
Can we remove this prop altogether now that we're not keeping separate KQL/filters per host/network?

Reply:
I missed that!!!